src/util.ts

```diff
@@ -9,12 +9,14 @@ import { isUndefined } from "util";
  *
  * @param coll the collection to convert into a docref array
  */
-export async function docRefArrayFromCollectionRef(coll: firestore.CollectionReference): Promise<Array<any>> {
+export async function docRefArrayFromCollectionRef(
+  coll: firestore.CollectionReference
+): Promise<Array<any>> {
   // Init array
   let insArr: firestore.DocumentReference[] = [];
   // Go through each doc and add their reference
-  return coll.get().then(snapshot => {
-    snapshot.forEach(element => {
+  return coll.get().then((snapshot) => {
+    snapshot.forEach((element) => {
```

Comment:
what's a snapshot?

Reply:
I'm honestly not that sure, from context, it seems like it's something like a "snapshot" of a collection??

Reply:
@morganbelous @henry-li-06 hey, a snapshot is actually a firebase term, and it's just like an index of a collection. So it's all the references of the documents, but not the data.